src/focus.js

```diff
@@ -0,0 +1,37 @@
+﻿import {customAttribute} from 'aurelia-templating';
+import {inject} from 'aurelia-dependency-injection';
+
+var afterAttachedSetFocusDelay = 50;
+
+@customAttribute('focus')
+@inject(Element)
+export class Focus {
+    constructor(element) {
+        this.element = element;
+
+        this.focusListener = (e => this.value = true).bind(this);
+        this.blurListener = (e => this.value = false).bind(this);
```

Comment:
You're right, the ```bind```s are useless.
For not returning the assigned value, you mean with something like this:
```this.focusListener = e => { this.value = true; };``` ?